docs/Governance.md

```diff
@@ -81,6 +81,7 @@ Responsibilities:
 
 In addition to having Maintainer privileges and responsibilities,
 a release manager is also responsible for generating releases in github.
+The release manager for this project must be a Microsoft full time employee.
```

Comment:
```suggestion
The release manager for this project must be a Microsoft full time employee in order to get Microsoft signed-binaries that Windows will load.
```
Explain why so it doesn't come across as an arbitrary fiat, there's actually a technical reason for this.